regression/cbmc/Function_Pointer_Init_No_Candidate/main.c

```diff
@@ -4,9 +4,7 @@ typedef int (*other_function_type)(int n);
 
 void foo(other_function_type other_function)
 {
-  // returning from the function call is unreachable -> the following assertion
-  //   should succeed
```

Comment:
So there is a semantic change for cases where function pointers are invalid / `NULL` / otherwise not properly defined?